python/generate-pypi-package.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Needs to be done beforehand:
+#cd /tmp
+#git clone https://github.com/CeetronSolutions/opm-common.git
+#cd opm-common
+#git checkout system-python-pypi
+
+yum-config-manager --add-repo \
+https://www.opm-project.org/package/opm.repo
+yum install -y boost169-devel boost169-static
+yum install -y blas-devel
+yum install -y suitesparse-devel
+yum install -y trilionos-openmpi-devel
+yum install -y cmake3
+yum install -y ccache
+yum install -y python3-devel
+python3 -m pip install pip --upgrade
+python3 -m pip install wheel pytest-runner
+python3 -m pip install twine
```

Comment:
Currently Python dependencies have been installed manually by @akva2 on the build server; unless he objects to continue that solution I suggest you send him the list of required Python packages (i.e. `twine`, `pytest-runner`...)`